web-registry/package.json

```diff
@@ -44,7 +44,8 @@
     "slick-carousel": "^1.8.1",
     "stream": "^0.0.2",
     "url-search-params-polyfill": "^8.1.0",
-    "web-components": "0.0.0"
+    "web-components": "0.0.0",
+    "yup": "^0.32.11"
```

Comment:
probably best to add it to web-components rather than web-registry

Reply:
done